lib/engine-util.js

```diff
@@ -0,0 +1,34 @@
+// Block colors
+const R = 'red';
```

Comment:
I think there should be array/object which lists all available colors in the game instead of this.